### PR TITLE
AccessControl: document FGAC licensing permissions

### PR DESCRIPTION
**Pull request by IevaVasiljeva**
**What this PR does / why we need it**:
Documents the new fine grained access control actions and roles for licensing.

**Which issue(s) this PR fixes**:
Docs for https://github.com/grafana/grafana-enterprise/issues/1859
Should be merged after [#1900](https://github.com/grafana/grafana-enterprise/pull/1900)

**Special notes for your reviewer**:
This is my first docs PR, so feedback is very welcome 🙏

### CODE REVIEW

File: docs/sources/enterprise/access-control/fine-grained-access-control-references.md [hunk 1]
```diff
@@ -27,11 +27,13 @@ The reference information that follows complements conceptual information about
...
+| Grafana Admin  | `fixed:permissions:admin:edit`<br>`fixed:permissions:admin:read`<br>`fixed:reporting:admin:edit`<br>`fixed:reporting:admin:read`<br>`fixed:users:admin:edit`<br>`fixed:users:admin:read`<br>`fixed:users:org:edit`<br>`fixed:users:org:read`<br>`fixed:ldap:admin:edit`<br>`fixed:ldap:admin:read`<br>`fixed:server:admin:read`<br>`fixed:settings:admin:read`<br>`fixed:settings:admin:edit`<br>`fixed:licensing:viewer`<br>`fixed:licensing:editor` | Allows access to resources which [Grafana Server Admin]({{< relref "../../permissions/_index.md#grafana-server-admin-role" >}}) has permissions by default. |
```
Comment (gamab) [inline docs/sources/enterprise/access-control/fine-grained-access-control-references.md:37]: We are not granting `fixed:licensing:viewer` anymore :slightly_smiling_face: 
```suggestion
| Grafana Admin  | `fixed:permissions:admin:edit`<br>`fixed:permissions:admin:read`<br>`fixed:reporting:admin:edit`<br>`fixed:reporting:admin:read`<br>`fixed:users:admin:edit`<br>`fixed:users:admin:read`<br>`fixed:users:org:edit`<br>`fixed:users:org:read`<br>`fixed:ldap:admin:edit`<br>`fixed:ldap:admin:read`<br>`fixed:server:admin:read`<br>`fixed:settings:admin:read`<br>`fixed:settings:admin:edit`<br>`fixed:licensing:editor` | Allows access to resources which [Grafana Server Admin]({{< relref "../../permissions/_index.md#grafana-server-admin-role" >}}) has permissions by default. |
```

File: docs/sources/enterprise/access-control/permissions.md [hunk 2]
```diff
@@ -67,6 +67,10 @@ The following list contains fine-grained access control actions.
...
 | `datasources:explore`      | n/a                                                                                     | Enable explore                                                                  |
+| `licensing:read`           | n/a                                                                                     | Read licensing information                                                      |
+| `licensing:update`         | n/a                                                                                     | Update the license token                                                        |
+| `licensing:delete`         | n/a                                                                                     | Delete the license token                                                        |
+| `licensing:reports:read`   | n/a                                                                                     | Get custom permission reports                                                   |
```
Comment (gamab) [inline docs/sources/enterprise/access-control/permissions.md:73]: I think @osg-grafana wanted us to end these with full stops.
```suggestion
| `licensing:read`           | n/a                                                                                     | Read licensing information.                                                      |
| `licensing:update`         | n/a                                                                                     | Update the license token.                                                        |
| `licensing:delete`         | n/a                                                                                     | Delete the license token.                                                        |
| `licensing:reports:read`   | n/a                                                                                     | Get custom permission reports.                                                   |
```

**Review by gamab — APPROVED** [review]
Left 2 comments :slightly_smiling_face: I think we can merge this, however I think we should wait for your front end pull requests to be merged first? So that we make this public once it's fully integrated frontend & backend. Wdyt?

**Comment by IevaVasiljeva** [comment]
> Left 2 comments 🙂 I think we can merge this, however I think we should wait for your front end pull requests to be merged first? So that we make this public once it's fully integrated frontend & backend. Wdyt?

Thanks for reviewing! And yes, makes sense to merge the frontend bits before the docs 👌

**Pull request merged**